packages/react-components/react-breadcrumb/src/components/BreadcrumbLink/useBreadcrumbLinkStyles.ts

```diff
@@ -70,12 +105,15 @@ export const useBreadcrumbLinkStyles_unstable = (state: BreadcrumbLinkState): Br
     styles[state.size],
     state.overflow && styles.overflow,
     state.current && currentSizeMap[state.size],
+    state.current && styles.current,
     state.root.className,
   );
 
   if (state.icon) {
-    state.icon.className = mergeClasses(styles.icon, state.icon.className);
+    state.icon.className = mergeClasses(iconStyles.base, iconStyles[state.size], styles.icon, state.icon.className);
   }
 
+  useLinkStyles_unstable(state as LinkState);
```

Comment:
Is it ok to use Link styles like that? Because Link state is a bit different from BreadcrumbLink state. From Link styles I need appearance and disabled states

Reply:
Yes, it's what is done in `Toolbar` AFAIR